### PR TITLE
Fix the text color and ad-big background color in Octotree extension

### PR DESCRIPTION
**Pull request by yungshenglu**
- Fix the text color in Octotree 
- Fix the background color of `ad-big` in the setting-views of Octotree

### CODE REVIEW

**Comment by poychang** [comment]
Except ad color, everything looks great. Wait for your opinion.

File: src/extensions/octotree-ext.scss [hunk 1]
```diff
@@ -12,26 +12,33 @@
...
-            }
-            .jstree-wholerow-hovered {
-                background: $dropdown-bg-active !important;
-            }
         }
         .octotree-settings-view {
             .octotree-view-body {
                 color: $bright-text-color !important;
             }
+            .octotree-ad-big {
```
[the remark this answers is not in the document]
Comment (yungshenglu) [reply src/extensions/octotree-ext.scss]: Sorry for the late response. I just thought that to keep the theme in the real dark 😄. It is okay to keep the original ad background color.

**Comment by yungshenglu** [comment]
Thanks for your advice. 👍

File: src/extensions/octotree-ext.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (poychang) [reply src/extensions/octotree-ext.scss]: Can you remove that and push into your own yungshenglu:develop branch and I can merge this PR 👍

[the remark this answers is not in the document]
Comment (yungshenglu) [reply src/extensions/octotree-ext.scss]: Okay, I will remove it. Thanks for reminding.

**Comment by TuKun33** [comment]
It's happening again
image

**Comment by TangXinGithub** [comment]
图片
It's happening again
what?

**Pull request merged**